types/client.go

```diff
@@ -184,40 +180,43 @@ type ImageLoadResponse struct {
 
 // ImagePullOptions holds information to pull images.
 type ImagePullOptions struct {
-	ImageID      string // ImageID is the name of the image to pull
-	Tag          string // Tag is the name of the tag to be pulled
-	RegistryAuth string // RegistryAuth is the base64 encoded credentials for the registry
+	Tag           string // Tag is the name of the tag to be pulled
```

Comment:
Tag should be part of argument. For example, "redis:latest" should be part of the argument. For now, we have to parse this on the client side, but in the future, this format should be daemon specific.